src/Repository/CommentsRepository.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repository;
+
+use App\Library\PDO;
+use App\Entity\CommentsEntity;
+
+class CommentsRepository
+{
+	//comment_frame.php.64
+	public static function setCommentByAll(int $id, string $post_body, string $posted_by, string $posted_to, mixed $date_added, int $post_id)
```

Comment:
Please make name of method more meaningful.

Please try to avoid passing more than 3 arguments to function consider something like "model" for modelling input data instead.

Also `mixed` type for date_added is also not the most fortunate as to me it looks like, it should be a datetime object.

Also keep in mind, your "date_added" with value mixed is ok, to accept `null` as a value, is DB structure ready for that?